src/main/java/com/blamejared/ambientenvironment/AmbientEnvironment.java

```diff
@@ -16,96 +16,98 @@
-            float scale = 16f;
+            final int newColor = waterColor.getColor(biome, posX, posZ);
+            final float scale = 16f;
             double value = ((NOISE_WATER.noiseAt(posX / scale, posZ / scale, false)));
-            double darkness = 0.3f;
-            value = curve(0, 1, remap(value, -((1 << levels) - 1), (1 << levels) - 1, 0, 1), 1) * darkness;
+            final double darkness = 0.3f;
+            value = curve(0, 1, remap(value, -((1 << levels) - 1), (1 << levels) - 1, 0, 1)) * darkness;
             return blend(newColor, 0, (float) (value));
         };
 //        BiomeColors.FOLIAGE_COLOR = (biome, pos) -> {
-//            int newColor = foliageColor.getColor(biome, pos);
-//            float scale = 8f;
+//            final int newColor = foliageColor.getColor(biome, pos);
+//            final float scale = 8f;
 //            double value = ((NOISE_FOLIAGE.getValue(pos.getX() / scale, pos.getZ() / scale)));
-//            double darkness = 1;
+//            final double darkness = 1;
 //            value = curve(0, 1, remap(value, -((1 << levels) - 1), (1 << levels) - 1, 0, 1), 1) * darkness;
 //            return blend(newColor, 0, (float) (value));
 //        };
     }
 
-    public static double remap(double value, double currentLow, double currentHigh, double newLow, double newHigh) {
+    public static double remap(final double value, final double currentLow, final double currentHigh, final double newLow, final double newHigh) {
         return newLow + (value - currentLow) * (newHigh - newLow) / (currentHigh - currentLow);
     }
 
-    private static float getRed(int hex) {
+    private static float getRed(final int hex) {
         return ((hex >> 16) & 0xFF) / 255f;
     }
 
-    private static float getGreen(int hex) {
+    private static float getGreen(final int hex) {
         return ((hex >> 8) & 0xFF) / 255f;
     }
 
-    private static float getBlue(int hex) {
+    private static float getBlue(final int hex) {
         return ((hex) & 0xFF) / 255f;
     }
 
-    private static float getAlpha(int hex) {
+    private static float getAlpha(final int hex) {
         return ((hex >> 24) & 0xff) / 255f;
     }
 
-    private static float[] getARGB(int hex) {
+    private static float[] getARGB(final int hex) {
         return new float[]{getAlpha(hex), getRed(hex), getGreen(hex), getBlue(hex)};
     }
 
-    private static int toInt(float[] argb) {
-        int r = (int) Math.floor(argb[1] * 255) & 0xFF;
-        int g = (int) Math.floor(argb[2] * 255) & 0xFF;
-        int b = (int) Math.floor(argb[3] * 255) & 0xFF;
-        int a = (int) Math.floor(argb[0] * 255) & 0xFF;
+    private static int toInt(final float[] argb) {
+        final int
+                r = (int) Math.floor(argb[1] * 255) & 0xFF,
+                g = (int) Math.floor(argb[2] * 255) & 0xFF,
+                b = (int) Math.floor(argb[3] * 255) & 0xFF,
+                a = (int) Math.floor(argb[0] * 255) & 0xFF;
```

Comment:
Please can you change this back to how it was before, making it a single declaration like this doesn't help anything and just makes the code harder to read.